am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -351,4 +371,24 @@ private DefaultPermissionGrant createDefaultPermissionGrant(String roleName,
             .attributePermissions(createPermissionsForAttribute(JsonPointer.valueOf(attribute), permissions, PUBLIC))
             .build();
     }
+
+
+
+    private Map<JsonPointer, SecurityClassification> getJsonPointerStringMap() {
```

Comment:
method could be renamed to something more meaningful - createSecurityClassifications? createPublicSecurityClassifications?

Reply:
done